client/me/purchases/manage-purchase/auto-renew-toggle/auto-renew-disabling-dialog/index.jsx

```diff
@@ -152,6 +152,19 @@ class AutoRenewDisablingDialog extends Component {
 		}
 	}
 
+	getGiftingCopy() {
+		const { siteDomain, translate } = this.props;
+
+		return translate(
+			"Did you know: You can allow site visitors to help cover the full cost of your site's plan and domain by turning on gifting in {{a}}Settings > General{{/a}}.",
+			{
+				components: {
+					a: <a href={ `/settings/general/${ siteDomain }` } />,
+				},
```

Comment:
```suggestion
			"{{strong}}Did you know:{{/strong}} You can allow site visitors to help cover the cost of your site's plan by enabling gift subscriptions in {{a}}Settings > General{{/a}}.",
			{
				components: {
					a: <a href={ `/settings/general/${ siteDomain }` } />,
					strong: <strong />,
				},
			}
```

Align copy with settings page + add some bold to the did you know part.

![Screenshot 2022-11-18 at 16-09-18 Purchase Settings ‹ Business Simple Test Site — WordPress com](https://user-images.githubusercontent.com/811776/202624248-b0764ae5-b2d1-45cb-93d8-f016b370fa9c.png)

Reply:
^ Not blocking, just a suggestion

Reply:
Thanks for adjusting it!